deploy/docker/cp-dts-status-checker/dts-status-checker.py

```diff
@@ -0,0 +1,145 @@
+
+
+def get_email_body(email_template_file):
+    body = {}
+    if os.path.exists(email_template_file):
+        with codecs.open(email_template_file,  "r", "utf-8") as html_file:
+            body = html_file.read()
+    else:
+        body = """<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
+    <html xmlns="http://www.w3.org/1999/xhtml">
+    <head>
+        <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
+        <style>
+            table,
+            td {{
+                border: 1px solid black;
+                border-collapse: collapse;
+                padding: 5px;
+            }}
+        </style>
+    </head>
+    <body>
+        <p>Dear user,<br>
+        *** This is a system generated email, do not reply to this email ***<br>
+        <b>New DTS status was detected:</b><br>
+        {}<br>
+        Best regards, 
+        EPM-AWS-Dev Platform<br>
+    </body>
+    </html>
+        """
+
+    return body
+
+
+def generate_table(dts_to_inform):
+    table = """<table>
+    <tr>
+		<td><b>ID</b></td>
+		<td><b>Status</b></td>
+		<td><b>Last heartbeat</b></td>
+	</tr>"""
+    for status in dts_to_inform:
+        table += "<tr>" + "<td>{}</td>".format(status["id"]) + "<td>{}</td>".format(
+            status["status"]) + "<td>{}</td>".format(status["heartbeat"]) + "</td>"
+
+    table += "</table>"
+    return table
+
+
+run_id = os.getenv('RUN_ID', default='0')
+pipeline_name = os.getenv('PIPELINE_NAME', default='pipeline')
+runs_root = os.getenv('CP_RUNS_ROOT_DIR', default='/runs')
+run_dir = os.getenv('RUN_DIR', default=os.path.join(runs_root, pipeline_name + '-' + run_id))
+log_dir = os.getenv('LOG_DIR', default=os.path.join(run_dir, 'logs'))
+pipeline_api = os.getenv('API')
+
+api = PipelineAPI(api_url=pipeline_api, log_dir=log_dir)
+
+logger = RunLogger(api=api, run_id=run_id)
+logger = TaskLogger(task='DTSStatusCheck', inner=logger)
+logger = LocalLogger(inner=logger)
+
+new_dts_statuses = fetch_current_dts_statuses()
+logger.info("Starting to check DTS")
+
+previos_dts_statuses = {}
+if os.path.exists(dts_file):
+    with open(dts_file) as json_file:
+        previos_dts_statuses = json.load(json_file)
+
+dts_to_inform = []
+
+for dts_id, new_dts_status in new_dts_statuses.iteritems():
+    current_status = new_dts_status["status"]
+    current_timestamp = new_dts_status["heartbeat"]
+    logger.info("DTS with id {} has status {}. Last heartbeat: {}".format(
+        dts_id, current_status, current_timestamp))
+    prev_dts_status = previos_dts_statuses.get(str(dts_id))
+    if not prev_dts_status or prev_dts_status["status"] != current_status:
+        logger.info("DTS with id {} has status: {}. Previous status: {}")
+        dts_to_inform.append(new_dts_status)
+    else:
+        logger.info("DTS with id {} has the same status: {}.")
```

Comment:
The same as [here](https://github.com/epam/cloud-pipeline/pull/2405/files#r773209059)